File: 2.0/csharp/contacts/updateDoNotContact.cs
using CallfireApiClient;
using CallfireApiClient.Api.Contacts.Model.Request;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        UpdateDncRequest request = new UpdateDncRequest()
        {
            Call = true,
            Text = false,
            Number = "12132000381"
        };
        client.DncApi.Update(request);
    }
}

File: 2.0/java/contacts/updateDoNotContact.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.contacts.model.request.UpdateDncRequest;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        UpdateDncRequest request = UpdateDncRequest.create()
            .call(true)
            .text(false)
            .number("'12132000381'")
            .build();
        client.dncApi().update(request);
    }
}
